drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -715,10 +712,7 @@ int nrf_wifi_if_stop_zep(const struct device *dev)
 	       sizeof(vif_info));
 
 	vif_info.state = NRF_WIFI_FMAC_IF_OP_STATE_DOWN;
-
-	memcpy(vif_info.ifacename,
-	       dev->name,
-	       strlen(dev->name));
+	vif_info.if_index = vif_ctx_zep->vif_idx;
```

Comment:
these changes are unrelated to commit log

Reply:
Included in a separate commit